rbac/management/migrations/0047_rolev2_rolebinding.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 4.2.10 on 2024-08-14 19:33
+
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        (
+            "management",
+            "0046_remove_workspace_the_combination_of_name_tenant_and_parent_must_be_unique__and_more",
+        ),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="RoleV2",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(default=uuid.uuid4, primary_key=True, serialize=False),
+                ),
+                ("is_system", models.BooleanField(default=False)),
+                (
+                    "v1_roles",
+                    models.ManyToManyField(related_name="v2_roles", to="management.role"),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Rolebinding",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(default=uuid.uuid4, primary_key=True, serialize=False),
+                ),
+                (
+                    "role",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="management.rolev2",
+                    ),
+                ),
+                (
+                    "v1_role",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="management.role",
```

Comment:
joining table has name `management_rolev2_v1_roles` and columns
`rolev2_id` and `role` - maybe `role` can be  named as `rolev1_id` ? 

There is table `management_rolebinding` with column `role` - but it is related to v2 - can it be also renamed to something like  `role_v2` ?

Also if names of role columns with prefixes or suffixes v1 and v2 will be consistent - it can help with readibility.

Reply:
Updated